src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -26,57 +26,45 @@ limitations under the License.
     <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
 
     <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST" type="ARRAY" entryType="INT16U" length="254" writable="false" optional="false">application launcher list</attribute>
```

Comment:
Again, none of the attribute names match the spec.  Followup for that, please.

Reply:
https://github.com/project-chip/connectedhomeip/issues/13213